tmt/schemas/test.yaml

```diff
@@ -22,10 +22,6 @@ properties:
   component:
     $ref: "/schemas/common#/definitions/one_or_more_strings"
 
-  # https://tmt.readthedocs.io/en/stable/spec/tests.html#contact
-  contact:
-    $ref: "/schemas/common#/definitions/one_or_more_strings"
-
```

Comment:
Tested simply using `tmt test show smoke`:

```
warn: /tests/core/smoke: - 'contact' does not match any of the regexes: '^extra-'
/tests/core/smoke
```

The `contact` key has to be defined here. And added to `story` and `plan` as well.

Reply:
`contact` is added to `test`, `story` and `plan`. Now `tmt tests show smoke` doesn't have the `warn:`. e.g.

```bash
(tmt) huanli@KVM:show$ tmt tests show smoke
/tests/core/smoke
                 summary Just a basic smoke test
                    test tmt --help
                    path /tests/core/smoke
               framework shell
                  manual false
                 require tmt
                duration 5m
                 enabled true
                  result respect
                     tag 'container' and 'virtual'
                    tier 0
```